src/python/zquantum/core/interfaces/backend_test.py

```diff
@@ -88,15 +88,15 @@ def test_run_circuit_and_measure_correct_indexing(self, backend):
         assert backend.number_of_circuits_run == 1
         assert backend.number_of_jobs_run == 1
 
-    @pytest.mark.parametrize("n_samples", [-1, 0, 100.2, 1000.0])
```

Comment:
What happens if a float is passed to this function?

Reply:
In the git hub thread referenced for this story, @max-radin says that it's better to catch issues with typing with mypy instead of at runtime. https://github.com/zapatacomputing/qe-qiskit/pull/76

> In general it's preferable to use type hints to discover type errors before execution rather than making assertions at runtime.

I'm just trying to follow that philosophy. :)

Reply:
Ok! Sounds good.